app/views/profiles/_form.html.erb

```diff
@@ -18,6 +18,39 @@
                    label: "Nickname",
                    help: "Unique. Reserved for future use on menadevs.com" %>
 
+
+
+<label>Skills</label>
+<select id="skill_list" name="skill_list[]" class="skills-input" class="js-example-basic-single" multiple="multiple">
+
+  User.where(weekly_subscriber: true).find_each do |user|
```

Comment:
sure will work on that, btw the weekly_subscriber part isn't actually part of the code, I pasted it by mistake, I just used it as a reference